pyfarm/agent/etc/agent.yml

```diff
@@ -63,6 +70,7 @@ agent_api_port: 50000
 # The location where the the agent should save its own
 # logging output to.
 agent_log: $agent_logs_root/agent.log
+agent_log: $agent_logs_root/supervisor.log
```

Comment:
> AFAIK, won't even work on Windows.

It can....but not under normal circumstances that I'd call normal but I'd thought I'd ask to be sure.  Thanks for clearing this up.